lib/main_common.pm

```diff
@@ -1141,22 +1141,8 @@ sub load_console_server_tests {
         loadtest "console/shibboleth";
     }
     if (!is_staging && (is_opensuse || get_var('ADDONS', '') =~ /wsm/ || get_var('SCC_ADDONS', '') =~ /wsm/)) {
-        loadtest "console/php_pcre";
         # TODO test on SLE https://progress.opensuse.org/issues/31972
         loadtest "console/mariadb_odbc" if is_opensuse;
-        if (is_leap("<15.4") || is_sle("<15-SP4")) {
-            loadtest "console/php7";
-            loadtest "console/php7_mysql";
-            loadtest "console/php7_postgresql";
-            loadtest "console/php7_timezone";
-        }
-        else {
-            loadtest "console/php7" unless is_sle;
-            loadtest "console/php8";
-            loadtest "console/php8_mysql";
-            loadtest "console/php8_postgresql";
-            loadtest "console/php8_timezone";
-        }
```

Comment:
I see you're removing the tests from main_common, but I don't see a shcedule for TW or Leap...

Reply:
You can see it now in the pull request description.